src/doc/man/cargo-package.md

```diff
@@ -81,8 +81,45 @@ There is no guarantee that the source code in the tarball matches the VCS inform
 
 {{#options}}
 
-{{#option "`-l`" "`--list`" }}
+{{#option "`-l`" "`--list`_[fmt]_" }}
 Print files included in a package without making one.
+
+Specifying an output format (rather than the default) is unstable and requires
+`-Zunstable-options`. Valid output formats:
+
+- `json` (unstable, requires `-Zunstable-options`): Emit machine-readable JSON
+  information about each file.
+  ```javascript
+  {
+    /* The Package ID Spec of the package. */
+    "path+file:///home/foo#0.0.0": {
+      /* Relative path in the archive file. */
+      "Cargo.toml.orig": {
+        /* An absolute path to the actual file content. */
+        "original_path": "/home/foo/Cargo.toml",
+        /* The source of the file.
+           - "existing" for file that exists on disk as-is.
+           - "generate" for file being generated during packaging
+           - "copied" for file copied from another location.
+             The associated path points to a resolved symlink target
+             or a normalized path outside the package root.
+        */
+        "source": "copied"
+      },
```

Comment:
I keep thinking "source path" when I see "source".   Wonder if there is a better name.  kind?  operation?  Not found one that I'm happy with.

Reply:
What about this?

```javascript
{
  "Cargo.toml": {
    "path": "Cargo.toml",
    "kind": "generated", 
  },
  "Cargo.toml.orig": {
    "path": "Cargo.toml",
    "kind": "on_disk",
  },
  "README.md": {
    "path": "../README.md",
    "kind": "on_disk",
  }
}
```

* Only "generated" and "on_disk".
  Assume that people don't need to know whether a file was copied from the other location.
  If they need,  the `path` field shows the it is different from the object key.
* Use relative path to the current working directory

Reply:
I think I'd just go with `coped` for all (they are all being copied from somewhere).  This also makes the verbage more consistent.

Speaking of verbage, should these be present tense (`generate`, `copy`)?